### PR TITLE
removed eval usage; and consistent severity level 

### PR DESCRIPTION
**Pull request by praveen-garg**
- [x] removed eval usage;  (vulnerable for external inputs)
- [x] severity levels are widespread but consistent now…; 
- [x] other formatting fixes

**test results:**

`================================================== 324 passed, 1 skipped in 6.51 seconds ==================================================`

### CODE REVIEW

File: python/foglamp/services/core/api/audit.py [hunk 1]
```diff
@@ -82,67 +84,52 @@ class Severity(IntEnum):
...
         if return_error is True:
-            raise web.HTTPNotFound(reason={"error": err_msg})
+            raise web.HTTPBadRequest(reason={"error": err_msg})
 
-        # Prepare the audit log method to call
-        audit_method = "audit." + str(severity).lower() + "(source, details)"
-
-        # Instantiate the audit log class
         audit = AuditLogger()
+        getattr(audit, str(severity).lower())(source, details)
```
Comment (pintomax) [inline python/foglamp/services/core/api/audit.py]: I see "await" is not in use with the code modification.

Comment (praveen-garg) [reply python/foglamp/services/core/api/audit.py]: great catch! thx

File: python/foglamp/services/core/api/audit.py [hunk 2]
```diff
@@ -49,100 +50,83 @@ class Severity(IntEnum):
...
 
-        # Get input data
-        json_data = await request.json()
+        payload = await request.json()
 
-        # Set items to store
-        severity = json_data.get(_SEVERITY_PARAM)
-        source = json_data.get(_SOURCE_PARAM)
-        details = json_data.get(_DETAILS_PARAM)
+        severity = payload.get("severity")
```
Comment (MarkRiddoch) [inline python/foglamp/services/core/api/audit.py]: I think this was better before rather than having repeated strings in the code.

Comment (praveen-garg) [reply python/foglamp/services/core/api/audit.py]: same I believe. 3 vars + each used at 2 places only;

**Pull request merged**